warehouse/index.js

```diff
@@ -463,6 +463,19 @@ function storeRudderEvent(utils, message, output, columnTypes, options) {
   ]
 */
 
+/*
+* Adds source and destination specific information into context
+* */
+function enhanceContextWithSourceDestInfo(context, metadata) {
+  if (!context || !metadata) {
```

Comment:
set empty context object {} if not present

Reply:
Done.